cmake_modules/FindHWLOC.cmake

```diff
@@ -72,13 +72,14 @@ if(HWLOC_FOUND)
 
   #===============================================================================
   # Import Target ================================================================
-  if(NOT TARGET HWLOC::HWLOC)
-    add_library(HWLOC::HWLOC INTERFACE IMPORTED)
-  endif(NOT TARGET HWLOC::HWLOC)
+  if(NOT TARGET hwloc)
+    add_library(hwloc INTERFACE IMPORTED)
+  endif(NOT TARGET hwloc)
 
-  set_property(TARGET HWLOC::HWLOC PROPERTY INTERFACE_LINK_LIBRARIES "${HWLOC_LIBRARY}")
-  set_property(TARGET HWLOC::HWLOC PROPERTY INTERFACE_INCLUDE_DIRECTORIES "${HWLOC_INCLUDE_DIR}")
-  set_property(TARGET HWLOC::HWLOC PROPERTY INTERFACE_COMPILE_OPTIONS "${HWLOC_DEFINITIONS}")
+  set_property(TARGET hwloc PROPERTY INTERFACE_LINK_LIBRARIES "${PC_HWLOC_LIB}")
```

Comment:
This may cause problems when using a Hwloc that is not system installed (not found by Pkg-config) but is findable by other means (e.g., user set HWLOC_ROOT) as this will mix the flags found from Pkg-config with the flags found from find_library.